### DISCUSSION TITLE
Continue adding custom recipes

### DISCUSSION THREAD

**Discussion by mani1232** [Zrips/CMI#7053] [Enhancement]
![recipes](https://user-images.githubusercontent.com/70480001/178141310-a7b074b6-b6a5-4c3d-891e-03ddc6b720e0.PNG)
Thank you for adding new tools for custom recipes, but there are more tools

**Comment by mani1232** [comment]
And add support for change particles from stonectter recipe

**Comment by mrfloris** [comment]
CMI 9.2.4.3 now has these at least:
<img width="477" alt="Screen-Shot 2022-09-25 at 18 15 27" src="https://user-images.githubusercontent.com/28841349/192153899-3eed7202-fd85-4966-a49b-f8e2a28e585b.png">
<img width="475" alt="Screen-Shot 2022-09-25 at 18 15 29" src="https://user-images.githubusercontent.com/28841349/192153900-d8951d3a-27ca-4ca1-8b6a-fbc849bd54e0.png">
<img width="476" alt="Screen-Shot 2022-09-25 at 18 15 32" src="https://user-images.githubusercontent.com/28841349/192153902-6229571f-ae07-4e54-b613-64df7f4e6678.png">
<img width="559" alt="Screen-Shot 2022-09-25 at 18 15 34" src="https://user-images.githubusercontent.com/28841349/192153904-f102e682-5954-43ad-9c43-45f7650e1b5c.png">
<img width="494" alt="Screen-Shot 2022-09-25 at 18 15 37" src="https://user-images.githubusercontent.com/28841349/192153905-188f2a2b-a280-4f84-84a7-34b4ffd02607.png">
<img width="546" alt="Screen-Shot 2022-09-25 at 18 15 39" src="https://user-images.githubusercontent.com/28841349/192153906-531fc19d-54a9-4a7c-9ca2-eec310d8f224.png">
<img width="540" alt="Screen-Shot 2022-09-25 at 18 15 43" src="https://user-images.githubusercontent.com/28841349/192153907-12c4f90e-fbf6-4a60-b899-02926f0b0d04.png">
<img width="482" alt="Screen-Shot 2022-09-25 at 18 15 46" src="https://user-images.githubusercontent.com/28841349/192153908-3dddcf34-9248-4724-8abf-cd25cdcf924e.png">

**Comment by mani1232** [comment]
thx
